README.md

```diff
@@ -101,31 +91,6 @@ $ kubectl patch mutatingwebhookconfiguration newrelic-metadata-injection-cfg --t
 
 Either certificate management choice made, the important thing is to have the secret created with the correct name and namespace. As long as this is done the webhook server will be able to pick it up.
 
-### 3) Enable the automatic Kubernetes metadata injection on your namespaces
-
-The injection is only applied to namespaces that have the `newrelic-metadata-injection` label set to `enabled`.
-
-```
-$ kubectl label namespace <namespace> newrelic-metadata-injection=enabled
-```
-
-## Development
-
-### Prerequisites
-
-For the development process [Minikube](https://kubernetes.io/docs/getting-started-guides/minikube) and [Skaffold](https://github.com/GoogleCloudPlatform/skaffold) tools are used.
-
-* [Install Minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/).
-* [Install Skaffold](https://github.com/GoogleCloudPlatform/skaffold#installation).
-
-Currently the project compiles with **Go 1.11.4**.
-
-### Dependency management
-
-[Go modules](https://github.com/golang/go/wiki/Modules) are used for managing dependencies. This project does not need to be in your GOROOT, if you wish so.
-
-Currently for K8s libraries it uses version 1.13.1. Only couple of libraries are direct dependencies, the rest are indirect. You need to point all of them to the same K8s version to make sure that everything works as expected. For the moment this process is manual.
-
 ### Configuration
 
 * Copy the deployment file `deploy/newrelic-metadata-injection.yaml` to `deploy/local.yaml`.
```

Comment:
Few lines below, where you talk about `imagePullPolicy`: I think that info is not correct. For local development, I think it is necessary to put the value of that option to `Never`.

Reply:
I never put it as `Never` locally, even before we had the images published. I just completely removed it. The default value is `IfNotPresent` and it works well with Skaffold, because the image will be present and not pulled.